### PR TITLE
Use SS more sparingly to help git grep.

### PR DESCRIPTION
**Pull request by schmmd**

### CODE REVIEW

**Comment by schmmd** [comment]
@rodneykinney, a comprimise

File: pipeline/src/main/scala/org/allenai/pipeline/ColumnFormats.scala [hunk 1]
```diff
@@ -13,8 +13,11 @@ import java.util.regex.Pattern
 trait ColumnFormats {
   private type SS[T] = StringSerializable[T]
```
Comment (jkinkead) [inline pipeline/src/main/scala/org/allenai/pipeline/ColumnFormats.scala]: Move to right before use in the implicit defs?

**Comment by rodneykinney** [comment]
I'm fine with getting rid of the alias altogether.  I only introduced it as a workaround, because scalariform would reformat the lines to exceed the line length of the style checker.

**Comment by schmmd** [comment]
@rodneykinney if you run into that problem (scalariform making your lines too long) let me or Jesse know.  There's a few simple tricks to learn about how to split lines of code.

**Comment by jkinkead** [comment]
The newer scalariform fork we started using in November is *much* easier to convince to keep lines short, since the default behavior for multi-line `defs` is to split it out further, rather than joining all on one line.

**Pull request merged**